backend/app/services/slack_notifier_service.rb

```diff
@@ -0,0 +1,17 @@
+class SlackNotifierService < Base
+  def initialize(message)
+    return if ENV['SLACK_TOKEN'].blank?
+      
+    @slack_client = Slack::Notifier.new ENV['SLACK_TOKEN']
+
+    @message = message
+  end
+
+  def call
+    return unless Rails.env.production?
```

Comment:
I think we can remove this so if we need we can try things in other environments